contracts/math/SafeMath.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.6.0;
```

Comment:
Note if we end up upgrading to 0.8.0 eventually, we won't need this!

Reply:
Yeah, upgrading to 0.8 is next in line.